charts/internal/seed-controlplane/charts/cloud-controller-manager/values.yaml

```diff
@@ -11,9 +11,8 @@ images:
   cloud-controller-manager: image-repository:image-tag
 resources:
   requests:
-    cpu: 11m
-    memory: 75Mi
+    cpu: 940m
```

Comment:
wow that is a lot cpu requests for the ccm. Is this reflecting the p10 percentile of the observed data?

Reply:
That was an error on my error. Thank you for noticing.